src/plugins/index_pattern_management/public/components/field_editor/components/field_format_editor/field_format_editor.test.tsx

```diff
@@ -26,15 +26,15 @@ const formatEditors = {
     ip: TestEditor,
     number: TestEditor,
   },
-  getById: jest.fn(() => TestEditor),
+  getById: jest.fn(() => () => Promise.resolve(TestEditor)),
 };
 
 describe('FieldFormatEditor', () => {
   it('should render normally', async () => {
     const component = shallow(
       <FieldFormatEditor
         fieldType="number"
-        fieldFormat={{} as DefaultFormatEditor}
```

Comment:
Apparently, this type was incorrect. Fixed casting in the test and in the file itself.